events/policy.go

```diff
@@ -0,0 +1,7 @@
+package events
```

Comment:
unused file?

Reply:
going to delete this one before merging, feel free to re-open if this one was needed for something!

Reply:
I think it was used before, but not anymore after resolving the most recent merge conflicts since it was added already somewhere else with a different name. Thanks for cleaning up!